src/shared/components/landing-image/LandingImage.js

```diff
@@ -1,38 +1,11 @@
 import React from "react";
-import { makeStyles } from "@mui/styles";
-import { useLocation } from "react-router-dom";
 
-import HomeLandingImg from "../../../assets/landing/home.jpg";
-import EventLandingImg from "../../../assets/landing/event.jpg";
-
-const useStyles = makeStyles({
-  bgImage: {
-    backgroundImage: `url(${HomeLandingImg})`,
```

Comment:
no I mean you can still use bgImage, but for the`backgroundImage` value, use the one from the caller. So:

`backgroundImage: `url(${LandingImg})`; This LandingImg variable will be given by the caller

Reply:
I have tried this since the start, but have no luck yet to make it work. First, this is outside the function component so the prop value can't be used here. Second, if I move this inside, it doesn't work. I will keep trying to find a way to do it but for now that's the cleanest solution I have.